src/ast/nodes/UnaryExpression.ts

```diff
@@ -55,7 +60,14 @@ export default class UnaryExpression extends NodeBase {
 	): LiteralValueOrUnknown {
 		if (path.length > 0) return UnknownValue;
 		const argumentValue = this.argument.getLiteralValueAtPath(EMPTY_PATH, recursionTracker, origin);
-		if (typeof argumentValue === 'symbol') return UnknownValue;
+		if (typeof argumentValue === 'symbol') {
+			if (this.operator === 'void') return undefined;
+			if (this.operator === '!') {
+				if (argumentValue === UnknownFalsyValue) return UnknownTruthyValue;
```

Comment:
Shouldn't this actually be "true"

Reply:
Oh, yeah, you're totally right! Thanks for pointing that out!